### PR TITLE
Ws docs

### PR DESCRIPTION
**Pull request by streflik**

### CODE REVIEW

**Comment by richxrich** [comment]
Hi guys,

Can you please address the questions from this thread regarding websockets:

https://github.com/mdyd-dev/platformos-documentation/issues/1440

Thanks

File: app/views/pages/use-cases/using-websockets.liquid [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      }
+    );
+  };
+}
+```
+
+### Step 5: Securing room subscription
+
+To make sure that only authorized users can access certain rooms (for example rooms for conversations where they are one of the pariticpants) we can use a partial that will be called when user tries to subscribe to a certain room.
+This file should be stored in `views/partials/channels/:channel_name/subscribed.liquid`. In this example we use code from the [pOS Marketplace Template](/developer-guide/pos-marketplace-template/core-concepts) and there channel is called `conversate`. This partial needs to return `true` string if current user is authorized to subscribe the room.
```
Comment (diana-lakatos) [inline app/views/pages/use-cases/using-websockets.liquid]: In `views/partials/channels/:channel_name/subscribed.liquid`, is the `:` needed?

**Review by diana-lakatos — CHANGES_REQUESTED** [review]

File: app/views/pages/use-cases/using-websockets.liquid (hunk 1 above)
Comment (Slashek) [reply app/views/pages/use-cases/using-websockets.liquid]: its a variable, that's why the `:`

**Review by z-x — APPROVED** [review]

**Review by diana-lakatos — APPROVED** [review]

File: app/views/pages/use-cases/using-websockets.liquid [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    function res = 'modules/chat/lib/commands/conversations/mark_unread', conversation: conversation, current_profile: current_profile
+  endif
+%}
+{% endraw %}
+```
+In this example, we are retrieving the conversation object and creating a message object to persist data in the database.
+
+### Other examples
+
+In the [pOS Marketplace Template](/developer-guide/pos-marketplace-template/core-concepts), we have also implemented a notifications functionality that uses WebSockets to inform user about unread messages. You can find it in the file `modules/chat/src/js/notifications.js`.
```
Comment (piotrze) [inline app/views/pages/use-cases/using-websockets.liquid]: This could be a link to the file in github repo `modules/chat/src/js/notifications.js`

**Review by piotrze — APPROVED** [review]

File: app/views/pages/use-cases/using-websockets.liquid (hunk 2 above)
Comment (diana-lakatos) [reply app/views/pages/use-cases/using-websockets.liquid]: Thank you! I added the link.

**Pull request merged**